pkg/contracts/src/OsmoticController.sol

```diff
@@ -96,46 +119,26 @@ contract OsmoticController is Initializable, OwnableUpgradeable, PausableUpgrade
         emit ProjectListCreated(list_);
     }
 
-    function createMimeToken(string calldata name, string calldata symbol, bytes32 merkleRoot)
+    function createMimeToken(string calldata name, string calldata symbol, bytes32 merkleRoot, uint256 roundDuration)
```

Comment:
I'm not completely sure we should let users to configure the round duration when creating mime tokens through our controller as we may end up with tokens with different duration when we intend to have an unique round period in our protocol :thinking: .

Maybe it would be better to have a configurable variable in the controller containing the fixed round duration